mephisto/core/operator.py

```diff
@@ -270,8 +270,7 @@ def parse_and_launch_run(
             raise e
 
         launcher = TaskLauncher(self.db, task_run, initialization_data_array)
-        launcher.create_assignments()
-        launcher.launch_units(task_url)
+        launcher.start(task_url)
```

Comment:
You shouldn't be updating the flow of how the operator tackles this just yet - your implementation should work under the separated flow (by only handling components that are necessary to run).